internal/pkg/template/upgrade/context.go

```diff
@@ -26,28 +26,15 @@ func NewContext(ctx context.Context, templateRef model.TemplateRef, objectsRoot
 
 	// Register existing IDs, so they will be reused
 	configs := search.ConfigsForTemplateInstance(projectState.RemoteObjects().ConfigsWithRowsFrom(targetBranch), instanceId)
-	for _, config := range configs {
-		// Config must exist and corresponding ID in template must be defined
```

Comment:
Vytvoril som funkciu/helper `iterateTmplMetadata`, kedze takyto `for` potrebujem na 2 miestach.